dspace/config/emails/internal_error

```diff
@@ -10,7 +10,7 @@
 ##
 ## See org.dspace.core.Email for information on the format of this file.
 ##
```

Comment:
This message is unsigned, but I'm inclined to let it be so, since it is an internal message and the instance is identified in the subject.